src/lang/fr/mod.rs

```diff
@@ -32,13 +34,28 @@ impl French {
     }
 }
 
+bitflags! {
+    /// Words that can be temporarily banned because of linguistic features.
+    ///(logical, numerical feature inconsistencies are already taken care of by DigitString)
+    struct Banned: u64 {
```

Comment:
Not happy with the name. If you have some better proposals, I'm hearing you!

Reply:
A few suggestions:
- Discarded
- Ignored
- Blocked
- Dropped
- Outliers